code/datums/ai/sanity/sanityloss_behaviors.dm

```diff
@@ -490,17 +487,7 @@
 	// Same as the above insanity, but they look for a target between moves.
 
 /datum/ai_behavior/insanity_wander/murder_wander/PreMoveCheck(datum/ai_controller/insane/murder/controller, mob/living/living_pawn)
-	for(var/mob/living/L in livinginview(9, living_pawn))
```

Comment:
Did you fuck this up?

Reply:
its doing the same checks here as FindEnemies and if we want to override targetting need to make it look at the proc that we are overriding.

Reply:
Otherwise it will stop wander behaviour even though there are no attackable enemies and be stuck there.